napari/tests/test_class_docstrings.py

```diff
@@ -0,0 +1,27 @@
+from napari.layers import (
+    Image,
+    Pyramid,
+    Markers,
+    Labels,
+    Shapes,
+    Vectors,
+)
+from napari import Viewer
+
+def test_docstrings():
+    for obj in (Image, Pyramid, Markers, Labels, Shapes, Vectors):
+        method = f'add_{obj.__name__.lower()}'
+        obj_doc = obj.__doc__
+        i = [idx for idx, section in enumerate(obj.__doc__.split('\n\n'))
+             if section.strip().startswith('Parameters')][0]
+        # We only check the parameters section, which is the first
```

Comment:
since the summary and return sections seem to follow a pattern, do we want to test these as well?

Reply:
I think this is fine for now as not all Layers docs have all of these, but in the future, we might standardize all of them so that they all have an `Attributes` and `Extended Summary` too at which point I agree. But if easy to add now and it doesn't hurt can do it now as well

Reply:
after speaking with @kne42 she made it clear that this was more about testing conformity of the summary and return sections in the docs in the viewer and not about the consistency between the docs in the viewer and the docs on the layers, so we said we'd wait to address in a follow-up PR